src/utility/copy.cpp

```diff
@@ -147,7 +125,15 @@ DuckdbCopy(PlannedStmt *pstmt, const char *query_string, struct QueryEnvironment
 		return false;
 	}
 
-	int token_start_offset = FindQueryTokenOffset(query_string, TO);
+	duckdb::string query_string_copy = std::string(query_string);
+
+	/* Quote filename so we can match it with query string*/
+	auto filename_quoted = quote_literal_cstr(copy_stmt->filename);
+	auto filename_quoted_end_pos = query_string_copy.find(filename_quoted) + strlen(filename_quoted);
```

Comment:
I'm not a fan of this. e.g. It'll break if if you have a query that contains this quoted filename. It might be okay for the 0.1.0, but I'd be much happier if the options string was built from the `List` of `DefElem`s in `copy_stmt->options`.